doc/development-setup.md

```diff
@@ -0,0 +1,48 @@
+# Development setup
+
+## Minimum supported Rust version
+
+The minimum supported Rust version for embedded-graphics is `1.40.0` or greater.
```

Comment:
The MSRV is also important for user of the library, so it should be included in the main readme.